db/migrate/20240202162322_alter_column_instructions_on_runs_to_allow_nil.rb

```diff
@@ -0,0 +1,5 @@
+class AlterColumnInstructionsOnRunsToAllowNil < ActiveRecord::Migration[7.1]
+  def change
+    change_column :runs, :instructions, :string, null: true
+  end
```

Comment:
In order to test for this case, I needed to flesh out the fixtures so there is another user (named Rob :) ) and this user has a conversation. In the process I discovered a small bug int he domain model: instructions should be optional. I went ahead and fixed that bug within this PR.